### ISSUE TITLE
Displayed release date off by one

### ISSUE THREAD

**Issue by SenorSmartyPants** [jellyfin/jellyfin#1486]
**Describe the bug**
Release date displayed in episode detail page is one day earlier than release date displayed on edit metadata page. 

I am in US CST.

**To Reproduce**
1. Go to tv episode page, notice date displayed
2. Click on 'edit metadata'
3. Scroll down to 'Release Date'

**Expected behavior**
Dates should match.

**Screenshots**

![image](https://user-images.githubusercontent.com/991618/59980341-293f9800-95ba-11e9-97da-61d4b8ccc445.png)

![image](https://user-images.githubusercontent.com/991618/59980343-38bee100-95ba-11e9-9c0e-6c966bcf32e0.png)

**System (please complete the following information):**
 - OS: Docker
 - Browser:Chrome
 - Jellyfin 10.3.5
 - Reverse proxy: no,

**Comment by sparky8251** [comment]
Does this only occur with Live TV? I ask because I see the show you are supplying as an example is likely being recorded with the Live TV functions.

If not, do you have any instances where this is an issue for Movies/Music as well?

And finally, what happens if you temporarily set your timezone on your computer to US Eastern? Id be curious if this is just a timezone issue or if its something "deeper."

**Comment by SenorSmartyPants** [comment]
I'm not using live TV functions right now, so this episode was not recorded with live TV functions.

Movie page seems to display Year and not the release date.

Changing TZ on JF client to EST did not correct displayed date. However changing TZ to UTC did get the correct date to display.

**Comment by sparky8251** [comment]
Ok. So perhaps it is still timezone related. Regardless, this will be a pain to fix properly...

Thanks much for the extra info!

**Comment by sparky8251** [comment]
Also, this should probably be moved to the web UI repo since it doesn't seem like a server issue to me (it reports the correct data as shown in the metadata manager).

Pinging @anthonylavado for moving.

**Comment by cvium** [comment]
I don't think this is really a bug. Tvdb returns the date 2019-06-18 in UTC with no timestamp, so that rightfully becomes 2019-06-18 00:00:00. JF-web always displays local time, which in this case is ~GMT-5, which becomes 2019-06-17.

**Comment by SenorSmartyPants** [comment]
I don't know what's happening in the database or code, but if no timestamp is returned then it should be treated as just a date and no timezone adjustments should be applied.
